src/server/memory/memory.h

```diff
@@ -195,6 +191,70 @@ class PlasmaBulkStore
   friend class SocketConnection;
 };
 
+
+class GPUBulkStore
+    : public BulkStoreBase<ObjectID, Payload>,
+      public std::enable_shared_from_this<GPUBulkStore>,
+      protected detail::ColdObjectTracker<ObjectID, Payload, GPUBulkStore> {
```

Comment:
We don't need the `ColdObjectTracker` at the moment, this is for the spilling feature under construction.